Components/FlightControl/MEVManager.cpp

```diff
@@ -0,0 +1,30 @@
+#include "MEVManager.hpp"
+#include "PBBRxProtocolTask.hpp"
+#include "CommandMessage.hpp"
+
+
+void MEVManager::OpenMEV() {
+    shouldMevBeOpen = OPEN;
+    PBBRxProtocolTask::SendPBBCommand(Proto::PBBCommand::Command::PBB_OPEN_MEV);
+}
+
+void MEVManager::CloseMEV() {
+    shouldMevBeOpen = CLOSE;
+    PBBRxProtocolTask::SendPBBCommand(Proto::PBBCommand::Command::PBB_CLOSE_MEV);
+}
+
+void MEVManager::HandleMEVTelemetry(Proto::TelemetryMessage& msg) {
+    if (shouldMevBeOpen == INDETERMINATE || !msg.has_mevstate()) {
+        // Do nothing
+        return;
+    }
+
+    Proto::MEVState mevState = msg.mevstate();
```

Comment:
Because its likely this stack obj doesnt get optimized out, either make this a reference or use .mevstate() directly